### ISSUE TITLE
UI Design

### ISSUE THREAD

**Issue by mjchamplin** [octalmage/minum#8]
I'm loving the minimal design of the app, and I'd love to refine the typography a bit and whatnot. Want me to work something up in Sketch?

**Comment by octalmage** [comment]
Yeah go for it! We still need the back and forward buttons. See #3.

**Comment by octalmage** [comment]
I think I want to have the top visited websites on the info screen, something like this:

![](http://cdn.techdows.com/wp-content/uploads/2014/08/Opera-Start-page-with-own-created-theme.png)

Idk, we have a bunch of real estate here:

![](https://cldup.com/3p8pyBHVD7.png)

I'm open to ideas!

**Comment by mjchamplin** [comment]
I still think it would be cool to tweak the UI a bit. A few initial thoughts, three years later: 

- Back and Forward buttons
- Slightly smaller title bar font (I'd bump it down 2pts and go from Light to Normal weight)
- Default window BG should be white, on websites without a declared `background-color` it's defaulting to gray
- The icon could probably use a bit more polish. 

You know, if you're interested. ;)

**Comment by octalmage** [comment]
Of course I'm interested! Got #3 for the back and forward buttons, not sure what to do there. Title bar font is easy, I'll make that happen. For the background color do you mean the title bar? The title bar is complicated, it captures the screen then attempts to grab the pixel color at the top of the screen and uses that. I've seen this not always work though. I have it set to read the pixel color on dom-ready for now, which means it might not always catch everything if things are loaded after dom-ready.

**Comment by octalmage** [comment]
And I love that icon! Still one of the best icons I've seen. I think it looks really nice in the dock. 

![screen shot 2018-06-21 at 9 55 15 pm](https://user-images.githubusercontent.com/142006/41755607-ce7dbcf4-759d-11e8-8705-88eb7030e391.png)

**Comment by mjchamplin** [comment]
Thank you for your kind words re: the icon! 

The title bar is great. I think I'll open a separate issue for the page-wide BG thing I'm seeing (with screenshots), since it's not really a UI issue. 

I'll think on the < > issue and follow up on #3 if I think of anything.

**Comment by octalmage** [comment]
Badass!
